style.css

```diff
@@ -123,3 +123,188 @@ p {
   line-height: 32px;
   text-align: center;
 }
+
+.card {
+  height: 350px;
+  background-color: #ebf0ee;
+  flex-direction: column;
+  justify-items: center;
+  justify-content: space-around;
+  align-content: center;
+  border-radius: 6px;
+  display: grid;
+  margin-top: 20px;
+  margin-right: 25px;
+  margin-left: 25px;
+}
```

Comment:
- [ ] For all cards, i.e the Languages, Frameworks and Sills cards, there should be a `border` of `1px solid #C1C7D0` and 
`border-radius` of `8px`. Kindly implement it. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/205454152-77c10c37-8ffc-4310-b33c-7602acd8c8e6.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/205454199-1bb337be-771a-4849-b025-0212a7efc9c1.png)